.github/workflows/main.yml

```
@@ -43,6 +43,13 @@ jobs:
       - name: Build with Maven
         run: ./mvnw -B -V -Dmaven.javadoc.skip=true -Ddb-user=$DB_USER -Ddb-password=$DB_PASSWORD -Ddb-schema=$DB_SCHEMA -Ptest clean package --file pom.xml
 
+      - name: Set up Java ${{ matrix.Java }} for run
+        uses: actions/setup-java@v2
```

actionlint (1.7.7)

the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)